fastlane/spec/fixtures/xcodeproj/automatic_code_signing.xcodeproj/project.pbxproj

```diff
@@ -392,6 +392,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				CODE_SIGN_STYLE = Manual;
```

Comment:
This shouldn't be here, should it?

Reply:
Oh man, my bad. It looks like the fastlane test suite modifies this file during a run and I accidentally committed it.

Reply:
Yep, we also noticed and have a issue/PR to either change this file or fix the problem. Super strange, probably Xcode doing stuff.